src/sass/components/_icons.scss

```diff
@@ -0,0 +1,41 @@
+@use "sass:map";
+@use "../abstracts" as abstracts;
+
+.yellow-icon {
+  stroke: map.get(map.get(abstracts.$highlight-colors, "yellow"), "default");
+}
+
+.cyan-icon {
+  stroke: map.get(map.get(abstracts.$highlight-colors, "cyan"), "default");
+}
+
+.pink-icon {
+  stroke: map.get(map.get(abstracts.$highlight-colors, "pink"), "default");
+}
```

Comment:
Voisiko näihin saada hoveri väritkin, jos ne on vaikka button:in tai Link:in sisällä ??

Reply:
Joo pitäs onnistua! Hyvä huomio.

Reply:
Esim. tuolla pages/_lists.scss on näin
```
  .edit-button {
    color: map.get(map.get(abstracts.$highlight-colors, "yellow"), "default");
  }

  .edit-button:hover {
    color: map.get(map.get(abstracts.$highlight-colors, "yellow"), "hover");
  }

  .delete-button {
    color: map.get(map.get(abstracts.$highlight-colors, "pink"), "default");
  }

  .delete-button:hover {
    color: map.get(map.get(abstracts.$highlight-colors, "pink"), "hover");
  }
```

Reply:
Sillain että nuo .delete-button vois korvata tolla pink-icon:illa

Reply:
Joo teen tänne ratkasun jossa on a- ja button-elementtien lapsena oleville ikonityyleille se hover. Korvaan nuo samalla.